scripts/sensor/04-group_average.py

```diff
@@ -175,19 +177,47 @@ def average_decoding(session):
         del contrast_score_stats, fname_out
 
 
+def get_config(
+    subject: Optional[str] = None,
+    session: Optional[str] = None
+) -> BunchConst:
+    cfg = BunchConst(
```

Comment:
here you don't need to pass subject I think as it's a group script but you need subjects and certainly session

Reply:
I've  now added `subjects = config.get_subjects()` to the `Bunch`; `session` is not necessary here due to the (weird?) way we're operating in `main()`. Should probably be refactored…